.github/workflows/test.yml

```diff
@@ -18,3 +18,17 @@ jobs:
       run: sudo modprobe ip_vs
     - name: Test
       run: sudo go test -v ./...
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go-version }}
```

Comment:
I think this isn't doing anything. Not sure what CI does in this case (it's an empty string in the logs).

Reply:
whoops fixed.